### PR TITLE
TRUNK-5334: Rename allergy.comment to allergy.comments

### PR DESCRIPTION
**Pull request by Anh3h**
Acceptance Criteria
x The "allergy.comment" column is renamed to "allergy.comments"
x Any existing data should be upgraded, invisibly to the end user
x The Allergy UI module should continue to work both before and after this change.
x The REST resource for allergies should continue to work both before and after this change, and the REST representation of allergy should remain unchanged

Ticket: https://issues.openmrs.org/browse/TRUNK-5334

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you add a link to the ticket as advised at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: api/src/main/resources/liquibase-update-to-2.0.xml [hunk 1]
```diff
@@ -8774,7 +8774,7 @@
             </column>
             <column name="allergen_type" type="varchar(50)">
             <constraints nullable="false"/></column>
-            <column name="comment" type="varchar(1024)"></column>
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase-update-to-2.0.xml]: Can you add a new changeset for renaming the column instead of modifying an existing one?

Comment (Anh3h) [reply api/src/main/resources/liquibase-update-to-2.0.xml]: Okay

**Comment by dkayiwa** [comment]
Did you see my topmost comment on this pull request?

**Comment by Anh3h** [comment]
Yea. I will add a link to the ticket.

**Comment by dkayiwa** [comment]
Why does your link include a filter?

**Comment by Anh3h** [comment]
I have removed the filter

File: api/src/main/java/org/openmrs/Allergy.java [hunk 2]
```diff
@@ -160,14 +160,14 @@ public void setSeverity(Concept severity) {
...
-		return comment;
+		return comments;
 	}
 	
 	/**
 	 * @param comment the comment to set
 	 */
-	public void setComment(String comment) {
-		this.comment = comment;
+	public void setComment(String comments) {
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/Allergy.java]: You need to deprecate getComment and setComment, add the new getComments and setComments and make sure the old getter/setter delegate to the new ones.

Comment (wluyima) [reply api/src/main/java/org/openmrs/Allergy.java]: In theory you don't even have to change comment to comments for the old getter and setter

**Comment by wluyima** [comment]
@Anh3h are you planning on addressing the comments on this PR?

**Comment by Anh3h** [comment]
Sorry but I am not sure, I am really tied up with my thesis and it's taking all of my time. I will close this PR and free up the issue for another developer.

**Comment by wluyima** [comment]
OK, thanks!

**Pull request closed without merging**